### ISSUE TITLE
Mild to moderate

### ISSUE THREAD

**Issue by LM-Ontologies** [obophenotype/human-phenotype-ontology#6487]
**Preferred term label:**
Mild to moderate

**Synonyms**

**Definition (free text, please give PubMed ID)**
Range between mild and moderate

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0012824

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
See recent comment for moderate to severe -- it would be confusing to users if we added X-to-Y terms for the existing severity terms, so I do not think this would be a good HPO term.
